### DISCUSSION TITLE
OBSPython Module

### DISCUSSION THREAD

**Discussion by gitagogaming** [upgradeQ/Streaming-Software-Scripting-Reference#28] [Q&A]
I was reading thru the docs a bit and saw mention of needing the OBSPython module to utilize this 'cheatsheet'.. but I've looked high and low and cannot locate this module anywhere.. 

was wondering if anyone could direct me to it?

**Comment by upgradeQ** [comment]
`obspython` module comes preinstalled with Software. You can't run it in `headless` state see [`Add a headless mode that allows full control via scripting / API`](https://ideas.obsproject.com/posts/16/add-a-headless-mode-that-allows-full-control-via-scripting-api) for more info.
To run examples in this cheatsheet follow these steps:
- Click `Tools` -> `Scripts` 
- Setup correct path, if you are on windows - you must install Python 3.6 version ( see [links](https://github.com/upgradeQ/OBS-Studio-Python-Scripting-Cheatsheet-obspython-Examples-of-API#links) section)
- Add a file by clicking on <kbd>+</kbd> button 
- Open logs to see if there were any errors

***Note:*** do not save any files where `url-text.py` and other default examples located - it will erase this directory in case of reinstalling the Software.
